deploy/docker/gaiahub.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
```

Comment:
Should the script include some extra strictness checks be turned on, such as `set -uo pipefail`?

Reply:
Ok. Done.